frontend/src/component/admin/users/InactiveUsersList/InactiveUsersList.tsx

```diff
@@ -0,0 +1,265 @@
+                    value: string;
+                }) => <RoleCell value={value} role={user.rootRole} />,
+                maxWidth: 120,
+            },
+            {
+                Header: 'Created',
+                accessor: 'createdAt',
+                Cell: DateCell,
+                width: 120,
+                maxWidth: 120,
+            },
+            {
+                id: 'last-login',
+                Header: 'Last login',
+                accessor: (row: any) => row.seenAt || '',
+                Cell: ({ row: { original: user } }: any) => (
+                    <TimeAgoCell
+                        value={user.seenAt}
+                        emptyText='Never'
+                        title={(date) => `Last login: ${date}`}
+                    />
+                ),
+                maxWidth: 150,
+            },
+            {
+                id: 'pat-last-login',
+                Header: 'PAT last used',
+                accessor: (row: any) => row.patSeenAt || '',
+                Cell: ({ row: { original: user } }: any) => (
+                    <TimeAgoCell
+                        value={user.patSeenAt}
+                        emptyText='Never'
+                        title={(date) => `Last used: ${date}`}
+                    />
+                ),
+                maxWidth: 150,
+            },
+            {
+                id: 'Actions',
+                Header: 'Actions',
+                align: 'center',
+                Cell: ({ row: { original: user } }: any) => (
+                    <InactiveUsersActionCell onDelete={openDelDialog(user)} />
+                ),
+                width: 200,
+                disableSortBy: true,
+            },
+        ],
+        [roles],
+    );
+    const initialState = useMemo(() => {
+        return {
+            sortBy: [{ id: 'createdAt', desc: true }],
+            hiddenColumns: ['username', 'email'],
+        };
+    }, []);
+
+    const { headerGroups, rows, prepareRow } = useTable(
+        {
+            columns: columns as any,
+            data: massagedData,
+            initialState,
+            autoResetHiddenColumns: false,
+            autoResetSortBy: false,
+            disableSortRemove: true,
+            disableMultiSort: true,
+            defaultColumn: {
+                Cell: TextCell,
+            },
+        },
+        useSortBy,
+        useFlexLayout,
+    );
+
+    return (
+        <PageContent
+            isLoading={loading}
+            header={
+                <PageHeader
+                    title={`Inactive users (${rows.length})`}
+                    actions={
+                        <>
+                            <Button
+                                variant='contained'
+                                color='primary'
+                                onClick={openDelInactiveDialog}
+                            >
+                                Delete all inactive users
+                            </Button>
```

Comment:
I think we should disable this if there are no inactive users.

Reply:
Sure.